libbeat/scripts/Makefile

```diff
@@ -103,6 +103,7 @@ crosscompile: $(GOFILES)
 
 .PHONY: check
 check: python-env ## @build Checks project and source code if everything is according to standard
+	@rm -rf ${GOPATH}/pkg # TODO go 1.9 supports `-source` flag in go vet, remove this in favor of the flag
```

Comment:
It's quiet intrusive that we remove the complete `pkg` folder. I wonder if this could have some side affects.

Reply:
I'll change this to go/pkg/linux_amd64/github.com/elastic/beats/ only, does it work for you?